mozdownload/scraper.py

```diff
@@ -697,14 +697,16 @@ def get_build_info(self):
         self.builds = parser.entries
         self.build_index = len(parser.entries) - 1
 
-        if self.build_number and \
-                ('build%s' % self.build_number) in self.builds:
```

Comment:
Same as for the original PR where I added this comment: https://github.com/mozilla/mozdownload/pull/513/files#r246667522 Please fix that.